packages/javascript-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@forgerock/javascript-sdk",
-  "version": "4.8.1",
+  "version": "4.8.2",
```

Comment:
Should this be 4.8.3 or will the changeset auto bump this? The current release in Github is already 4.8.2

Reply:
it would get bumped, i just manually made this up to date because i had to do a manual publish for the last release